subprojects/scala/src/main/java/org/gradle/api/tasks/scala/ScalaDoc.java

```diff
@@ -133,6 +136,28 @@ public void setTitle(@Nullable String title) {
         this.title = title;
     }
 
+    /**
+     * Returns the amount of memory allocated to this task.
```

Comment:
Can you add an example of what is expected here (1G, 256MB, etc)?

Reply:
This should be resolved now